github-actions/scan/src/configuration-model-customizer.ts

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+
+const ADDITIONAL_EXCLUDES: string[] = ['**/.sechub-gha/**']
+
+/**
+ * Adds additional exclude patterns to the provided JSON data.
+ * 
+ * If a 'data' section is present the additional exclude patterns will be added to each entry's 'excludes' array (sources and binaries).
+ * In case the 'data' section is not present, the deprecated 'codeScan.excludes' will be checked instead.
+ * If no sources or binaries are specified at all, no default exclude patterns will be added.
+ * 
+ * @param sechubConfigJson - The JSON data to be updated.
+ */
+export function addAdditonalExcludes(sechubConfigJson: any): void {
+    handleDataSection(sechubConfigJson.data);
+    handleLegacyCodeScanSection(sechubConfigJson.codeScan);
+}
+
+function handleDataSection(dataSection: any) {
+    if (dataSection === undefined) {
```

Comment:
this will not treat `dataSection === null`. was this intended?

```suggestion
    if (dataSection == undefined) {
```